tests/misc_env/install_prereq.py

```diff
@@ -173,6 +184,41 @@ def enable_rhel_rpms(ceph, distro_ver):
         )
 
 
+def enable_rhel_eus_rpms(ceph, distro_ver):
+    """
+    Setup cdn repositories for rhel systems
+    Args:
+        distro_ver: distro version details
+        ceph: ceph object
+    """
+
+    eus_repo = {"7": ("7.7", ["rhel-7-server-eus-rpms"])}
+
+    ceph.exec_command(
+        sudo=True,
+        cmd="subscription-manager release --set={distro}".format(
+            distro=eus_repo.get(distro_ver[0])[0]
+        ),
+        long_running=True,
+    )
+
+    for repo in eus_repo.get(distro_ver[0])[1]:
+        ceph.exec_command(
+            sudo=True,
+            cmd="subscription-manager repos --enable={r}".format(r=repo),
+            long_running=True,
+        )
+
+    ceph.exec_command(sudo=True, cmd="yum clean all", long_running=True)
+
+    ceph.exec_command(
+        sudo=True,
+        cmd="yum --releasever={distro} update".format(
```

Comment:
We should not have to provide `--releasever` here to Yum. `subscription-manager release --set` should take care of this globally for all `yum` operations.

Reply:
done, removed